### DISCUSSION TITLE
scrollbar - issue

### DISCUSSION THREAD

**Discussion by Kens-Jaswal** [saadeghi/daisyui#2231] [General]
my website browser and other content scrollbar is always visible, how I can hide that globally.
I am using tailwindcss.com and daisyui.com with svelte..

I saw in tailwind and daisyui components like (Table with pinned-rows and pinned-cols) scroll is always hide and when I scroll that appears..

I wants to use this thing in my website, kindly help ...pls

**Comment by saadeghi** [comment]
It's not easy to help when you don't provide a URL or a reproduction repo or a live example (Tailwind play, StackBlitz, etc)
